apps/main/src/components/PagePool/PoolDetails/ChartOhlcWrapper/PoolActivity.tsx

```diff
@@ -80,9 +80,19 @@ const PoolActivity = ({
             <EventTitle>{eventOption === 'TRADE' ? t`Swap` : t`Action`}</EventTitle>
             <TimestampColumnTitle>{t`Time`}</TimestampColumnTitle>
           </TitlesRow>
-          <ElementsContainer>
+          <ElementsContainer minHeight={chartExpanded ? 548 : 330}>
             {eventOption === 'TRADE' ? (
```

Comment:
It might be better to split this into separate conditions, rather than using hard-to-read nested ternary operators